php.ini-production

```diff
@@ -1764,20 +1764,20 @@ ldap.max_links = -1
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=0
+;opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+;opcache.enable_cli=1
```

Comment:
I agree with @staabm. Either should be mentioned that this is experimental or it should be enabled, as PHP is emerging also for lon-running scripts which might benefit from opcache in CLI.